packages/office-addin-debugging/src/commands.ts

```diff
@@ -2,12 +2,27 @@
 // Licensed under the MIT license.
 
 import * as commander from "commander";
+import * as fs from "fs";
 import { logErrorMessage, parseNumber } from "office-addin-cli";
 import * as devSettings from "office-addin-dev-settings";
 import { OfficeApp, parseOfficeApp } from "office-addin-manifest";
-import { AppType, parseAppType, parseDebuggingMethod, startDebugging } from "./start";
+import { AppPlatform, AppType, parseAppPlatform, parseAppType, parseDebuggingMethod, startDebugging } from "./start";
 import { stopDebugging } from "./stop";
 
+function determineManifestPath(platform: AppPlatform, dev: boolean): string {
+    let manifestPath = "";
+    if (process.env.npm_package_config_manifest_location) {
```

Comment:
please use a blank line between variables and code. 
please use a bank line before the return line.